### PR TITLE
fix(Facebook): profile username

### LINKED ISSUE #6331: Facebook | facebook.com
### Steps to Reproduce

1. https://www.facebook.com/oww.lala.549/
2. Check Discord Activity
3. Check this with privacy mode enabled

### Expected Behavior

Viewing Profile

### Actual Behavior

only Facebook, and the logo itself

### Page URL

https://www.facebook.com/oww.lala.549/

### Screenshots

1. https://prnt.sc/fBL3yTkCJimC
2. https://prnt.sc/q-YZjkK5GJh3

### Operating System

Windows 10

### Browser and Version

Brave: Version 1.39.122 Chromium: 102.0.5005.115 (Official Build) (64-bit)

### Additional Comments

Previous topic: #6072

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

Fixed the profile username being shown viewing someone's profile & privacy mode when viewing someone's profile
Resolves #6331
## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/F/Facebook/dist/metadata.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
 		"vi_VN": "Kết nối với bạn bè, người thân và những người khác bạn biết. Chia sẻ hình ảnh và video, gửi tin nhắn và nhận các thông báo."
 	},
 	"url": "www.facebook.com",
-	"version": "3.2.12",
+	"version": "3.2.13",
```
Comment (EncryptedDev) [inline websites/F/Facebook/dist/metadata.json:25]: It seems like functionality for profiles didn't exist prior to this. If so, you are adding a feature so this should be a minor bump
```suggestion
	"version": "3.3.0",
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/F/Facebook/dist/metadata.json (hunk 1 above)
Comment (Kaushik9371) [reply websites/F/Facebook/dist/metadata.json]: profile <shrishidahiya121>

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Comment by Kaushik9371** [comment]
profile : sonakshi.ahlawat.58
email ID : rosy@icloud.com

File: websites/F/Facebook/dist/metadata.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
 		"vi_VN": "Kết nối với bạn bè, người thân và những người khác bạn biết. Chia sẻ hình ảnh và video, gửi tin nhắn và nhận các thông báo."
 	},
 	"url": "www.facebook.com",
```
Comment (Kaushik9371) [inline websites/F/Facebook/dist/metadata.json:24]: "www.facebook.com/login/shristidahiya121

**Review by Kaushik9371 — COMMENTED** [review]
[``](url;https://www.facebook.com/login/sonakshi.ahlawat.58)

File: websites/F/Facebook/dist/metadata.json (hunk 2 above)
Comment (DarkVillager) [reply websites/F/Facebook/dist/metadata.json:24]: ok

**Pull request merged**